packages/vscode-extension/src/service/TranslationKeys.ts

```diff
@@ -173,3 +173,18 @@ export const UPDATE_OBJECT = {
 	PROCESS_CANCELED: 'UPDATE_OBJECT_PROCESS_CANCELED',
 	UPDATING: 'UPDATE_OBJECT_UPDATING',
 };
+
+export const CREATE_FILE = {
+	QUESTIONS: {
+		CHOOSE_SUITESCRIPT_TYPE: 'CREATE_FILE_CHOOSE_SUITESCRIPT_TYPE',
```

Comment:
Not very important but we are usually following format CREATE_FILE_QUESTIONS_CHOOSE_SUITESCRIPT_TYPE. The same applies for MESSAGES

Reply:
Fixed